src/main/java/org/quiltmc/loader/impl/util/SystemProperties.java

```diff
@@ -23,6 +23,7 @@
 	public static final String GAME_VERSION = "quilt.gameVersion";
 	public static final String REMAP_CLASSPATH_FILE = "quilt.remapClasspathFile";
 	public static final String LAUNCHER_NAME = "quilt.launcherName";
+	public static final String PRINT_MOD_RESOLVING = "quilt.debug.mod_resolving";
```

Comment:
I'd probably rename the field to `DEBUG_MOD_RESOLVING`

Reply:
Sure!